docs/user-guide/src/bmo/live_updates_servicing.md

```diff
@@ -0,0 +1,98 @@
+# Live updates (servicing)
+
+Live updates (servicing) enables baremetal-operator to conduct certain actions
+on already provisioned BareMetalHosts. These actions currently include:
+
+- [configuring BIOS settings](./firmware_settings.md)
```

Comment:
nit: we should rather call them "firmware settings" in the metal3 context (same below)